File: LeetCode/9. Palindrome Number/Solution.java
// https://leetcode.com/problems/palindrome-number/

class Solution {
    public boolean isPalindrome(int x) {
        if(x<0)
            return false;
        int rev=0; int y=x;
        while(y>0)
        {
            int a=y%10;
            rev=rev*10+a;
            y=y/10;
        }
        if(rev==x)
            return true;
        else 
            return false;
        
    }
}

File: LeetCode/9. Palindrome Number/Solution.py
# https://leetcode.com/problems/palindrome-number/

class Solution:
    def isPalindrome(self, x: int) -> bool:
      
        # Method - 1 ( Convert int to str and compare )
        temp = str(x)
        return temp == temp[::-1]
    
        # Method - 2 ( Reverse the number and compare )
        ori = x
        tempcopy = 0
        while(x > 0 ):
            tempcopy = tempcopy*10 + x % 10
            x = x//10
        return tempcopy == ori
      
        
